axelor-account/src/main/java/com/axelor/apps/account/service/ScaleServiceAccountImpl.java

```diff
@@ -6,30 +6,19 @@
 import com.axelor.apps.account.db.MoveLine;
 import com.axelor.apps.base.db.Company;
 import com.axelor.apps.base.db.Currency;
-import com.axelor.apps.base.service.ScaleService;
-import com.axelor.apps.base.service.app.AppBaseService;
-import com.google.inject.Inject;
+import com.axelor.apps.base.service.ScaleServiceImpl;
 import java.math.BigDecimal;
 
-public class ScaleServiceAccountImpl implements ScaleServiceAccount {
-
-  private static final int DEFAULT_SCALE = AppBaseService.DEFAULT_NB_DECIMAL_DIGITS;
-
-  protected ScaleService scaleService;
-
-  @Inject
-  public ScaleServiceAccountImpl(ScaleService scaleService) {
-    this.scaleService = scaleService;
-  }
+public class ScaleServiceAccountImpl extends ScaleServiceImpl implements ScaleServiceAccount {
 
   @Override
   public BigDecimal getScaledValue(BigDecimal value) {
-    return scaleService.getScaledValue(value);
+    return super.getScaledValue(value);
   }
 
   @Override
   public BigDecimal getScaledValue(BigDecimal value, int customizedScale) {
-    return scaleService.getScaledValue(value, customizedScale);
+    return super.getScaledValue(value, customizedScale);
   }
```

Comment:
Because this is now an extension there is no recreate these.

Reply:
Resolved